packages/grid_client/src/modules/models.ts

```diff
@@ -635,6 +636,10 @@ class FilterOptions {
   @Expose() @IsOptional() @Transform(({ value }) => NodeStatus[value]) @IsEnum(NodeStatus) status?: NodeStatus;
   @Expose() @IsOptional() @IsString() region?: string;
   @Expose() @IsOptional() @IsBoolean() healthy?: boolean;
+  @Expose() @IsOptional() @IsBoolean() planetary?: boolean;
```

Comment:
can we revisit this section, can we have planetary inside features only ?

Reply:
No, This's needed as we set up the features with those filters and I can't really push them directly to the features from the dashboard as users might deploy from the grid client scripts